src/kvstore/kvstore_dist.h

```diff
@@ -228,17 +228,18 @@ class KVStoreDist : public KVStoreLocal {
           RunContext rctx, Engine::CallbackOnComplete cb) {
         // convert to ps keys
         size_t size = recv_buf.shape().Size();
-
+        int dtype = recv_buf.dtype();
+        int elem_size = mshadow::mshadow_sizeof(dtype);
```

Comment:
would `num_bytes` be a better name?

Reply:
Haha, that's what I had originally. I changed it from `num_bytes` to `elem_size` today evening! I changed it because I thought num_bytes was conveying the meaning of total number of bytes as opposed to the number of bytes for each element. Do you still think `num_bytes` is better?